### ISSUE TITLE
Feature Request: Option to use absolute rather than relative dates

### ISSUE THREAD

**Issue by fncll** [mdirolf/crosshare#363]
There are various places that show dates (pub date of puzzles, comment dares) as relative. It would be awesome if we could choose to use absolute/real dates instead.

**Comment by mdirolf** [comment]
Everywhere there's a relative date there is a title tag that uses the absolute date - so you should be able to hold your mouse over any relative date to see a pop-up with the absolute one 😊

**Comment by fncll** [comment]
Is there any way to achieve this on a tablet/phone?

**Comment by mdirolf** [comment]
@fncll good point - I guess I assumed titles were viewable on mobile by touch+hold but it appears they aren't. Changing this issue title to reflect the new scope of work
